src/app.ts

```diff
@@ -30,7 +30,7 @@ const cli = new Cli({
         reg.setHomeserverToken(AppServiceRegistration.generateToken());
         reg.setAppServiceToken(AppServiceRegistration.generateToken());
         reg.setSenderLocalpart("slackbot");
-        reg.addRegexPattern("users", `@${config.username_prefix}.*`, true);
+        reg.addRegexPattern("users", `@${config.username_prefix}.*:${config.homeserver.server_name}`, true);
```

Comment:
Hm, shouldn't the server name be regex-escaped too?

Reply:
Oh maybe. Hopefully people don't have bizarre looking servernames, and if they do, they will know to fix it.